packages/remediations/src/schema.js

```diff
@@ -26,8 +26,43 @@ export default (container) => ({
                         {
                             type: validatorTypes.REQUIRED
                         }]
+                    }],
+                    nextStep: ({ values }) => values.multiple ? 'actions' : 'review'
+                },
+                {
+                    name: 'actions',
+                    title: 'Review and edit actions',
+                    fields: [{
+                        name: 'manual-resolution',
+                        component: 'review-actions'
+                    }],
+                    nextStep: ({ values }) => values['manual-resolution'] ? 'issue-resolution-0' : undefined
```

Comment:
should not it be `: 'review'` ? (the same below)

Reply:
it should but it does not exist yet, it is just something to display to avoid error

Reply:
sure, it should, I left it there by an accident, thank you